Create rule S6589: Dockefiler should only have one ENTRYPOINT and CMD instruction (#1749)
SonarSource/rspec: rules/S6589/docker/rule.adoc

> Dockerfile can contain at most one ENTRYPOINT and one CMD instruction, otherwise, only the last one will have an effect.

Reviewer: ```suggestion
Dockerfile should contain at most one ENTRYPOINT and one CMD instruction, because only the last one will have an effect.
```
can -> should - because Docker syntax allows many ENTRYPOINT & CMD instructions
otherwise -> because - it sounds simpler for me
